dvc/remote/config.py

```diff
@@ -156,3 +155,18 @@ def get_default(self, level=None):
         return self.config.get(
             Config.SECTION_CORE, Config.SECTION_CORE_REMOTE, level=level
         )
+
+    def default_remote_set(self):
+        """
+        Checks if default remote config is present.
+        Args:
+            rconfig: a remote config
+
+        Returns:
+            True if the default remote config is set, else False
+        """
+        try:
+            self.get_default()
+            return True
+        except ConfigError:
+            return False
```

Comment:
Let's not have it here, as no one is using it. Let's keep this helper in dvc/external_repo.py